playbooks/customize.yml

```diff
@@ -4,6 +4,10 @@
   gather_facts: no
 
   vars_prompt:
+    - name: streisand_ssh_private_key
+      prompt: "Enter the path to your SSH private key? Press enter for default "
```

Comment:
"Enter the path to your SSH private key, or press enter for default" reads a bit better in my opinion.  Either way the question mark can be dropped.

Reply:
Thanks! fixed in https://github.com/jlund/streisand/pull/970/commits/ef529d198bcf6643a39f5b6bc445c21ada698f48